packages/amplication-client/src/style/mixin.scss

```diff
@@ -410,3 +416,11 @@ $active-indicator-width: 4px;
   line-height: 18px;
   color: #f5f6fa;
 }
+
+@mixin regular-12-gray {
+  @include font-base();
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #a3a8b8;
+}
```

Comment:
please do not add mixin for fonts unless it is part of the bigger initiative to align the texts and fonts in the system